projects/plugins/jetpack/extensions/plugins/publicize/index.js

```diff
@@ -64,8 +63,6 @@ const PublicizeSettings = () => {
 
 	return (
 		<PostTypeSupportCheck supportKeys="publicize">
-			<TwitterThreadListener />
-
 			<JetpackPluginSidebar>
 				<PublicizePanel enableTweetStorm={ true }>
```

Comment:
I think we could remove the `enableTweetStorm` prop too, but we could pick that up in another change.

Reply:
Yeah, there are many other things we could remove I think. Is removing traces of Twitter elements something that was on y'all's to-do list? I looked for a tracking issue, couldn't find any, but I assume that's something that came up when we deprecated Publicize to Twitter?

Reply:
Yes it came up, but I think there was a hope that we might be able to bring it back too, so we only removed the essential parts.

I've added a task to our backlog to tidy this up

Reply:
> I've added a task to our backlog to tidy this up

To close the loop on this, this is being removed in #33723.